README.md

```diff
@@ -1,2 +1,46 @@
-# okta-spring-security-oauth2
-okta-spring-security-oauth2
+okta-spring-security
+====================
+
+This repo is still a little raw, and will be cleaned up before merging to master.
+Of interest:
+
+- okta-spring-security-oauth is a Spring Boot starter use to configure an implicit flow access token validation.
```

Comment:
This seems to imply it's a starter and not a library. If it's a starter, I think we should name it `okta-spring-boot-starter`.